<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>GBAnalMessageBox_Dialog</class>
 <widget class="QDialog" name="GBAnalMessageBox_Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>621</width>
    <height>548</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Analysis is running, please wait patiently ...</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QDialog
{
	background-color: rgb(43, 43, 43);
}
QComboBox
{
	border-radius: 3px;
}
QComboBox::hover
{
	background-color: rgb(244, 244, 244);
}
QComboBox::drop-down
{
	width: 20px;
	border-left: 1px solid rgb(192, 192, 192);
	border-top-right-radius: 3px;
	border-bottom-right-radius: 3px;
}
QComboBox::down-arrow
{
	image:url(ui/ico/DownArrow.png);
	width: 10px;
}
QGroupBox
{
	background-color: rgb(128, 128, 128);
	color: rgb(255, 255, 255);
}
QLabel
{
	background-color: rgb(255, 255, 255);
	color: rgb(0, 0, 0);
	border-radius: 3px;
}
QPushButton
{
	background-color: rgb(255, 255, 255);
	border-radius: 3px;
}
QPushButton::hover
{
	background-color: rgb(144, 200, 246);
}
QPushButton:pressed
{
    padding-left: 3px;
    padding-top: 3px;
}
QPushButton::disabled
{
	background-color: rgb(160, 160, 160);
	color: rgb(80, 80, 80);
}
QTextBrowser
{
	background-color: rgb(43, 43, 43);
	color: rgb(255, 255, 255);
}
QToolButton
{
	border-radius: 5px;
	color: rgb(255, 255, 255);
}
QToolButton:hover
{
	background-color: rgba(255, 255, 255, 20%);
}
QToolButton:pressed
{
	background-color: rgba(0, 0, 0, 20%);
	border-top: 1px solid rgba(0, 0, 0, 50%);
	border-left: 1px solid rgba(0, 0, 0, 50%);
	padding: 0;
}</string>
  </property>
  <property name="sizeGripEnabled">
   <bool>false</bool>
  </property>
  <layout class="QGridLayout" name="gridLayout" rowstretch="1,14,0,0" columnstretch="1,0">
   <property name="sizeConstraint">
    <enum>QLayout::SetDefaultConstraint</enum>
   </property>
   <property name="leftMargin">
    <number>2</number>
   </property>
   <property name="topMargin">
    <number>2</number>
   </property>
   <property name="rightMargin">
    <number>2</number>
   </property>
   <property name="bottomMargin">
    <number>2</number>
   </property>
   <property name="spacing">
    <number>2</number>
   </property>
   <item row="1" column="0" rowspan="2" colspan="2">
    <widget class="QTextBrowser" name="MCMessageBox_textBrowser">
     <property name="font">
      <font>
       <family>Segoe UI</family>
       <pointsize>9</pointsize>
       <italic>false</italic>
       <bold>false</bold>
      </font>
     </property>
    </widget>
   </item>
   <item row="3" column="0" colspan="2">
    <widget class="QGroupBox" name="groupBox_4">
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>34</height>
      </size>
     </property>
     <property name="title">
      <string/>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <property name="spacing">
       <number>4</number>
      </property>
      <property name="leftMargin">
       <number>4</number>
      </property>
      <property name="topMargin">
       <number>4</number>
      </property>
      <property name="rightMargin">
       <number>4</number>
      </property>
      <property name="bottomMargin">
       <number>4</number>
      </property>
      <item>
       <widget class="QLabel" name="STime_label">
        <property name="minimumSize">
         <size>
          <width>80</width>
          <height>22</height>
         </size>
        </property>
        <property name="text">
         <string>Start Timeï¼š</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
        </property>
        <property name="margin">
         <number>6</number>
        </property>
        <property name="indent">
         <number>1</number>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="StartTimeDay_label">
        <property name="minimumSize">
         <size>
          <width>70</width>
          <height>22</height>
         </size>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="margin">
         <number>2</number>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="StartTimer_label">
        <property name="minimumSize">
         <size>
          <width>70</width>
          <height>22</height>
         </size>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="margin">
         <number>2</number>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="horizontalSpacer">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>112</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QPushButton" name="Stop_pushButton">
        <property name="enabled">
         <bool>true</bool>
        </property>
        <property name="sizePolicy">
         <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="minimumSize">
         <size>
          <width>56</width>
          <height>22</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true">QPushButton
{
	background-color: rgb(255, 0, 0);
	color: rgb(255, 255, 255);
	border-radius: 3px;
}
QPushButton::disabled
{
	background-color: rgb(160, 160, 160);
	color: rgb(80, 80, 80);
}</string>
        </property>
        <property name="text">
         <string>Stop</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="OK_pushButton">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="minimumSize">
         <size>
          <width>56</width>
          <height>22</height>
         </size>
        </property>
        <property name="text">
         <string>OK</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="0" column="0" colspan="2">
    <widget class="QGroupBox" name="groupBox">
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>34</height>
      </size>
     </property>
     <property name="title">
      <string/>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <property name="leftMargin">
       <number>4</number>
      </property>
      <property name="topMargin">
       <number>4</number>
      </property>
      <property name="rightMargin">
       <number>4</number>
      </property>
      <property name="bottomMargin">
       <number>4</number>
      </property>
      <item>
       <widget class="QLabel" name="ExportMessage_label">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="minimumSize">
         <size>
          <width>80</width>
          <height>22</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>160000</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="text">
         <string>Export message to:</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
        </property>
        <property name="margin">
         <number>6</number>
        </property>
        <property name="indent">
         <number>1</number>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QComboBox" name="ExportFileFormat_comboBox">
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>22</height>
         </size>
        </property>
        <item>
         <property name="text">
          <string>Text File Format</string>
         </property>
        </item>
       </widget>
      </item>
      <item>
       <spacer name="horizontalSpacer_2">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeType">
         <enum>QSizePolicy::Fixed</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>180</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QToolButton" name="SaveText_toolButton">
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>22</height>
         </size>
        </property>
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
